pages/instructorAssessmentStatistics/instructorAssessmentStatistics.sql

```diff
@@ -1,20 +1,23 @@
--- BLOCK assessment_stats
```

Comment:
This query is still in use in line 84 of instructorAssessmentStatistics.js (the score stats CSV).

Reply:
Good catch, thanks! Fixed in a83518ec7a0ab8627258a3e39e5e288856223925